# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmFrameIter.cpp
# Commit: 9111720170a2
# Full Hash: 9111720170a283e7c3801856d5c113a2f17caf20
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-05-06 09:50:52
# Regressor Bug: 1642412
# File Overlap Count: 7
# Description:
#   Bug 1762171 - wasm: Fix cranelift build for table optimizations. r=yury
#   
#   This commit adds back the builtins that were removed in
#   bug 1642412.
#   
# ==============================================================================

diff -r 2340f31f2559 -r 9111720170a2 js/src/wasm/WasmFrameIter.cpp
--- a/js/src/wasm/WasmFrameIter.cpp	Thu May 05 21:49:46 2022 +0000
+++ b/js/src/wasm/WasmFrameIter.cpp	Thu May 05 21:49:46 2022 +0000
@@ -1578,14 +1578,16 @@
       return "call to native table.fill function";
     case SymbolicAddress::ElemDrop:
       return "call to native elem.drop function";
-    case SymbolicAddress::TableGetFunc:
+    case SymbolicAddress::TableGet:
       return "call to native table.get function";
     case SymbolicAddress::TableGrow:
       return "call to native table.grow function";
     case SymbolicAddress::TableInit:
       return "call to native table.init function";
-    case SymbolicAddress::TableSetFunc:
+    case SymbolicAddress::TableSet:
       return "call to native table.set function";
+    case SymbolicAddress::TableSize:
+      return "call to native table.size function";
     case SymbolicAddress::RefFunc:
       return "call to native ref.func function";
     case SymbolicAddress::PreBarrierFiltering: